ocaml/xapi/repository.ml

```diff
@@ -44,7 +55,7 @@ let introduce ~__context ~name_label ~name_description ~binary_url ~source_url
              )
      ) ;
   create_repository_record ~__context ~name_label ~name_description ~binary_url
-    ~source_url ~update ~gpgkey_path
+    ~source_url ~update ~gpgkey_path ~origin
```

Comment:
The value of `update` needs to be ensured as `true` for the bundle repo. I think xapi can tolerate it even when it is specified as `false` in API call parameters.